jest.config.js

```diff
@@ -1,12 +1,14 @@
 module.exports = {
   preset: "react-native",
+  setupFiles: ["<rootDir>jest-ts-auto-mock-config.ts"],
```

Comment:
Does this need to have the ```/```  between the dir and file name  so it would look like ```setupFiles: ["<rootDir>/jest-ts-auto-mock-config.ts"],```?

Reply:
When I googled it it does look like the leading `/` is required.  I've added it in.